src/main/webapp/quizFunctions.js

```diff
@@ -23,10 +23,10 @@ document.getElementById("start_quiz").onclick = function() {
                 Object.keys(players_ids_and_photos).forEach(function(key) {
                     fetch('/get-image?blobKey=' + players_ids_and_photos[key]).then((pic) => {
                     let button_for_picture = document.createElement("BUTTON");
-                    button_for_picture.name="user_picture";
+                    button_for_picture.name= "user_picture";
```

Comment:
if we're going to add the space on the right (which we should), we should add it on the right also.